ModKit/UI/UI+Browser.cs

```diff
@@ -11,6 +12,10 @@ namespace ModKit {
 
     public static partial class UI {
         public class Browser {
+            public enum sortDirection {
+                Asc = 1,
```

Comment:
Please spell out identifier names.  I will land this but please fix this or I will and avoid it in the future.